docker/fabric-tools/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM hyperledger/fabric-tools:2.4
 
+# avoid the shell swallowing errors
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+
 # Install curl and netcat
-RUN apk update && \
-  apk add curl netcat-openbsd vim libc6-compat && \
+RUN apk --no-cache add curl netcat-openbsd vim libc6-compat && \
```

Comment:
we no longer need the apk update?

Reply:
This is because previously this didn't go through Hadolint (our Docker linter), but now that it does I needed to address [this warning](https://github.com/hadolint/hadolint/wiki/DL3019). It should actually make the image smaller (since we didn't remove the cache afterwards) so that's a plus.